packages/varlet-ui/src/picker/docs/en-US.md

```diff
@@ -447,6 +447,7 @@ function handleChange(values, indexes) {
 | `cascade` | Whether to enable cascade mode | _boolean_ | `true` |
 | `option-height` | The height of the option | _string \| number_ | `44` |
 | `option-count` | Number of visible options | _string \| number_ | `6` |
+| `columns-num`  | Number of visible columns | _string \| number_ | `Infinity` |
```

Comment:
remember to add version `***3.3.11***`

Reply:
<img width="709" alt="image" src="https://github.com/user-attachments/assets/93ad835a-fe4b-4cea-9dec-f82cb0883984">